receptorctl/receptorctl/cli.py

```diff
@@ -104,7 +104,7 @@ def status(ctx):
     ads = status.pop('Advertisements', None)
     if ads:
         print()
-        print(f"{'Node':<{longest_node}} Service   Type       Last Seen            Tags")
+        print(f"{'Node':<{longest_node}} Service   Type       Last Seen            Work Commands                Tags")
```

Comment:
If you've got a ton of work commands defined, this table may not print the headers very pretty.

But I also don't feel like implementing complicated ascii table drawing (given that this is just a CLI tool for humans).

Reply:
maybe we can just put Work Commands to the far right, so if it is really long it won't crash in to the Tags column

Reply:
![image](https://user-images.githubusercontent.com/214912/107292861-8494f700-6a38-11eb-903d-353497e8bb0f.png)